telink: b92: change the section in retention.

haiwentelink opened the pull request:
- change the west.yml .
- move ramcode_ble to RAM_ILM_N .

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/430c104d68cb63ae23f91c7a1cda6e506822fb45 | https://github.com/telink-semi/hal_telink/commit/1d22efe9ab5b371248aae09cd41ff4db25406f43 (develop) | [telink-semi/hal_telink@430c104d..1d22efe9](https://github.com/telink-semi/hal_telink/compare/430c104d68cb63ae23f91c7a1cda6e506822fb45..1d22efe9ab5b371248aae09cd41ff4db25406f43) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

haiwentelink committed:
telink: b92: change the section in retention.

- change the section in retention.
- change the suspend para.
- improve stablity of ble.
- change the west.yml.

Signed-off-by: Haiwen Xia <haiwen.xia@telink-semi.com>
